incubator/openmpi/values.yaml

```diff
@@ -0,0 +1,30 @@
+# Default values for openmpi.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+mpiWorker:
+  number: 5
+  podManagementPolicy: Parallel
+  image:
+    repository: cheyang/openmpi
+    tag: 3.0.0
+    pullPolicy: IfNotPresent
+  sshPort: 22
+  resources: {}
+  privileged: true
```

Comment:
This line needs to move either up or down, so `resources` is 'connected' to the example resources.